dealership/customers.py

```diff
@@ -1,11 +1,20 @@
 class Person(object):
     def __init__(self, first_name, last_name, email):
-        pass
+        self.first_name = first_name
+        self.last_name = last_name
+        self.email = email
 
 
 class Customer(Person):
-    pass
-
+    def __init__(self, first_name, last_name, email):
+        super(Customer, self).__init__(first_name, last_name, email)
+ 
+    def is_employee(self):
+        return False
 
 class Employee(Person):
-    pass
+    def __init__(self, first_name, last_name, email):
```

Comment:
technically, you don't need `__init__` here since you got the same parameters as `Person`